src/containers/App/App.js

```diff
@@ -5,30 +5,40 @@ import { connect } from 'react-redux';
 import { getMovies } from '../../actions';
 import API from '../../utils/api';
 import { apiKey } from '../../utils/api-key';
+import Movie from '../Movie/Movie'
 
 class App extends Component {
-  constructor(props) {
-    super(props);
-
-  }
```

Comment:
Agree we don't need to constructor, at least not yet. I ran into the same thing building out ToDo Box where app is stateless.

Reply:
I'm hoping that we may not need it at all, it feels a little unnecessary to be honest... but we'll see